packages/modules/viewer-react/src/components/app-ui/widgets/BasicNavigationWidget.tsx

```diff
@@ -114,7 +127,16 @@ export function BasicNavigationWidget({
       items.push(
         ToolbarHelper.createToolbarItemFromItemDef(
           10,
-          CoreTools.walkViewCommand
+          new ToolItemDef({
```

Comment:
Looks good, i just searched and had found LookAndMoveTool in \core\frontend\src\tools\ViewTool.ts
Would it be possible to use that, or do you need to redefine the tool item like this?

Reply:
Just tried using the tool, surprisingly doesn't fit with the typing of the function even though it extends the Tool class